### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by arturojgt** [kubilus1/autoortho#381] [Q&A] [answered]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 11
- XPlane Version: XP12 12.06r3
- AutoOrtho Version [e.g. 0.6.1]
- Filesystem type NTFS
- Latest DOKAN

### question

I had installed it in a previous XP-12 installation and worked fine. I made a new installation but this time not. I tried DOKAN and WINFSP but the error persisted.

Log:
INFO:autoortho:AutoOrtho version: 0.6.0-1
ERROR:config_ui:XPlane install directory '' is not a directory.
ERROR:config_ui:No installed scenery detected!

![Captura de pantalla 2023-09-09 180218](https://github.com/kubilus1/autoortho/assets/29168872/c0d5a51a-a0b9-451f-8d61-1ca99ef83106)

**Comment by chezhj** [comment, marked as answer]
Does this help https://github.com/kubilus1/autoortho/issues/375#issuecomment-1705667767

**Reply by arturojgt** [reply]
Thank you, your suggestion solved my problem.

**Reply by chezhj** [reply]
That's great, please mark as Answered
